crates/kornia-core/src/tensor.rs

```diff
@@ -653,6 +653,36 @@ where
         self.map(|x| x.powi(n))
     }
 
+    /// Apply the power function to the tensor data.
+    ///
+    /// # Arguments
+    ///
+    /// * `n` - The power to raise the tensor data to.
+    ///
+    /// # Returns
+    ///
+    /// A new tensor with the data raised to the power.
+    pub fn powf(&self, n: T) -> Tensor<T, N, A>
```

Comment:
the point of having the kornia-core-ops was to start reducing operators in the tensor definition. Does it make sense to have `TensorOps` trait that implements all this `map` based operators ?

Reply:
I understand. I was unsure how would you like to go about moving the old operators over and the powf function was so close to the existing operators, that I thought I would rather put it with them. I can move them over to a TensorOps trait if we are ready for that.

Reply:
yes, can do step by step and iterate on the api